OCPBUGS-6615: OSP: Add note about rgw_max_attr_size in OSP17 (#55816)
openshift/openshift-docs: modules/installation-osp-enabling-swift.adoc

If Swift is present and you want to use it, you must enable access to it. If it is not present, or if you do not want to use it, skip this section.
====

[IMPORTANT]
====

> Early versions of {rh-openstack} 17.0 set `rgw_max_attr_size` parameter of Ceph RGW to 256. This setting causes issues with uploading container images to the {product-title} registry. `rgw_max_attr_size` should be set to 1024. Before installation please check if your {rh-openstack} is affected by this problem and if so reconfigure Ceph RGW.

Reviewer: ```suggestion
Early versions of {rh-openstack} 17.0 set the `rgw_max_attr_size` parameter of Ceph RGW to 256 ${UNITS}. This setting causes issues with uploading container images to the {product-title} registry. You must set the value of `rgw_max_attr_size` to [at least?] 1024. 

Before installation, check if your {rh-openstack} deployment is affected by this problem. If it is, reconfigure Ceph RGW.
```

Can we be more specific than "early?"

Author: "Early" in this case means that [1] is not yet fixed, but there's a consensus that it should be. I'm not sure how to handle this, maybe we just drop "early" and assume every deployment is affected?

Should I replace `${UNITS}` with something here? In this case it's string length, so "characters"?

[1] https://bugzilla.redhat.com/show_bug.cgi?id=2167161

Reviewer: 1. Fine by me. 

2. Yeah, and "characters" is fine. We just need to avoid ambiguity as much as possible for translation's sake.

Author: Alright, should be fixed now.

If Swift is present and you want to use it, you must enable access to it. If it is not present, or if you do not want to use it, skip this section.
====

[IMPORTANT]
====

> Early versions of {rh-openstack} 17.0 set the `rgw_max_attr_size` parameter of Ceph RGW to 256 characters. This setting causes issues with uploading container images to the {product-title} registry. You must set the value of `rgw_max_attr_size` to at least 1024 characters.

Reviewer: ```suggestion
{rh-openstack} 17 sets the `rgw_max_attr_size` parameter of Ceph RGW to 256 characters. This setting causes issues with uploading container images to the {product-title} registry. You must set the value of `rgw_max_attr_size` to at least 1024 characters.
```

> Before installation, check if your {rh-openstack} deployment is affected by this problem. If it is, reconfigure Ceph RGW.

Reviewer: ```suggestion
Before installation, check if your {rh-openstack} deployment is affected by this problem. If it is, reconfigure Ceph RGW.
```
Is there documentation that succinctly describes how to check for this issue?

Author: I don't have any and IIRC Ceph allows you to set options in various places, so this might be too complicated to explain here.
